support-frontend/assets/pages/digital-subscription-checkout/components/orderSummary/orderSummaryStyles.js

```diff
@@ -53,7 +63,7 @@ export const imageContainer = css`
   align-items: flex-start;
   width: calc(100%-30px);
   padding: 15px 10px 0 15px;
-  background-color: #63717A;
+  background-color: #F6F6F6;
```

Comment:
Is #F6F6F6 in @guardian/src-foundations/palette?

Reply:
And this. Thanks George